core/templates/dev/head/components/activity_tiles_infinity_grid_directive.html

```diff
@@ -1,6 +1,7 @@
 <script type="text/ng-template" id="components/activityTilesInfinityGrid">
   <div infinite-scroll="showMoreActivities()" infinite-scroll-distance="1"
-       infinite-scroll-disabled="endOfPageIsReached || searchResultsAreLoading">
+       infinite-scroll-disabled="endOfPageIsReached || searchResultsAreLoading"
+       style="max-width: 856px; margin-left: auto; margin-right: auto;">
```

Comment:
Can you make this a CSS class (oppia-activities-infinity-grid-container perhaps) and put its definition at the bottom of this file?